catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/TransformersCommand.java

```diff
@@ -0,0 +1,151 @@
+
+    // output strings
+
+    static final String ACTIVE_TRANSFORMERS_HEADER = "Active Transformers: ";
+
+    static final String NO_ACTIVE_TRANSFORMERS = "There are no active transformers";
+
+    private static final String LINE = "------------------------";
+
+    // Transformer Properties
+
+    private static final String MIME_TYPE = "mime-type";
+
+    private static final String SCHEMA = "schema";
+
+    private static final String NOT_AVAILABLE = "N/A";
+
+
+
+    @Override
+    protected Object executeWithSubject() throws Exception {
+
+        // ServiceReferences are needed to get property information from InputTransformers
+        List<ServiceReference<InputTransformer>> serviceReferences =
+                (List<ServiceReference<InputTransformer>>) bundleContext.getServiceReferences(
+                        InputTransformer.class,
+                        "(id=*)");
+
+        List<TransformerProperties> transformersProperties = serviceReferences.stream()
+                .map(TransformerProperties::new)
+                .collect(Collectors.toList());
+
+        int activeTransformers = transformersProperties.size();
+
+        if (activeTransformers == 0) {
+            console.printf("%s%n%n", NO_ACTIVE_TRANSFORMERS);
+            return null;
+        }
+
+        console.printf("%n%s%d%n%s%n%n", ACTIVE_TRANSFORMERS_HEADER, activeTransformers, LINE);
+
+        Iterator<TransformerProperties> tpIterator = transformersProperties.iterator();
+        TransformerProperties tp;
+
+        while (tpIterator.hasNext()) {
+            tp = tpIterator.next();
+            console.printf("%s", tp.printProperties());
+
+            if (tpIterator.hasNext()) {
+                console.printf("%n%s%n", StringUtils.repeat(LINE, 3));
+            }
+            console.printf("%n");
+        }
+
+        return null;
+    }
+
+    private static class TransformerProperties {
+
+        private String id;
+
+        private String schema;
+
+        private List<String> mimeTypes;
+
+        public TransformerProperties(ServiceReference ref) {
+            this.id = getTransformerProperty(ref, Constants.SERVICE_ID);
+            this.schema = getTransformerProperty(ref, SCHEMA);
+            this.mimeTypes = getTransformerMimeProperty(ref);
+        }
+
+        public String printProperties() {
+            StringBuilder s = new StringBuilder(MessageFormat.format(
+                    "{0}: {1}\n\n\t{2}: {3}\n\t{4}s: {5}\n",
+                    Constants.SERVICE_ID,
+                    id,
+                    SCHEMA,
+                    schema,
+                    MIME_TYPE,
+                    mimeTypes.remove(0)));
+
+            for (String mimeType : mimeTypes) {
+                s.append("\t\t    ")
+                        .append(mimeType)
+                        .append("\n");
+            }
+            return s.toString();
+        }
+
+        private String getTransformerProperty(ServiceReference ref, String property) {
```

Comment:
✏️  You could rename this method to `getTransformerPropertyStringValue` or something like that to clarify that it will only work for properties that are strings, as opposed to, for example, the mime type string list property.